debug/src/check-props.js

```diff
@@ -2,6 +2,25 @@ const ReactPropTypesSecret = 'SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED';
 
 let loggedTypeFailures = {};
 
+/**
+ * Reset the history of which prop type warnings have been logged.
+ */
+export function resetPropWarnings() {
+	loggedTypeFailures = {};
+}
```

Comment:
I added this helper for use in tests to reset the history of logged warnings. This is currently re-exported from the `preact/debug` package, so this is a minor API addition.

Is there a preferred alternative way of creating this kind of helper for use in tests?

Reply:
Sounds good to me 👍